### PR TITLE
cgen: fix for_in map of fixed_array

### PR DESCRIPTION
**Pull request by yuyi98**
This PR fixes for_in map of fixed_array.

- Fix for_in map of fixed_array.
- Add tests.

```vlang
fn test_for_in_map_of_fixed_array() {
	mut rets := []string{}
	m := map{'aa': [1, 2]!, 'bb': [3, 4]!, 'cc': [5, 6]!}

	for k, v in m {
		println('$k, $v')
		rets << '$k, $v'
	}
	assert rets[0] == 'aa, [1, 2]'
	assert rets[1] == 'bb, [3, 4]'
	assert rets[2] == 'cc, [5, 6]'
}

fn test_for_mut_in_map_of_fixed_array() {
	mut rets := []string{}
	mut m := map{'aa': [1, 2]!, 'bb': [3, 4]!, 'cc': [5, 6]!}

	for k, mut v in m {
		println('$k, $v')
		rets << '$k, $v'
	}
	assert rets[0] == 'aa, [1, 2]'
	assert rets[1] == 'bb, [3, 4]'
	assert rets[2] == 'cc, [5, 6]'
}
```

### CODE REVIEW

File: vlib/v/gen/c/cgen.v [hunk 1]
```diff
@@ -1490,28 +1500,21 @@ fn (mut g Gen) for_in_stmt(node ast.ForInStmt) {
...
-		g.stmts(node.stmts)
-		if node.key_type == table.string_type && !g.is_builtin_mod {
-			// g.writeln('string_free(&$key);')
-		}
-		if node.label.len > 0 {
-			g.writeln('\t${node.label}__continue: {}')
 		}
-		g.writeln('}')
-		if node.label.len > 0 {
-			g.writeln('\t${node.label}__break: {}')
```
Comment (spytheman) [inline vlib/v/gen/c/cgen.v:1512]: Why was that deleted?
what will replace it?

Comment (spytheman) [reply vlib/v/gen/c/cgen.v:1512]: We may need additional regression tests for `break outer` and `continue outer`, where outer is a label on an outer for in loop.

Comment (yuyi98) [reply vlib/v/gen/c/cgen.v:1512]: It also remove return and just like other cases using following process.
```vlang
        ...
	g.stmts(node.stmts)
	if node.label.len > 0 {
		g.writeln('\t${node.label}__continue: {}')
	}
	g.writeln('}')
	if node.label.len > 0 {
		g.writeln('\t${node.label}__break: {}')
	}
```

Comment (spytheman) [reply vlib/v/gen/c/cgen.v:1512]: Thanks.
